lib/std/child_process.zig

```diff
@@ -777,36 +776,36 @@ fn windowsCreateProcess(app_name: [*:0]u16, cmd_line: [*:0]u16, envp_ptr: ?[*]u1
 /// Caller must dealloc.
 /// Guarantees a null byte at result[result.len].
 fn windowsCreateCommandLine(allocator: *mem.Allocator, argv: []const []const u8) ![]u8 {
```

Comment:
This one actually should use `Buffer`, or otherwise fulfill the duties of the API. The return type should be changed to `![:0]u8`